packages/orchestration/src/util/filtering.ts

```diff
@@ -0,0 +1,59 @@
+import { supportedAzureFilterThresholds } from '../orchestration-types.js';
+import type {
+  AzureContentSafety,
+  AzureContentSafetyFilterConfig,
+  InputFilteringConfig,
+  OutputFilteringConfig
+} from '../client/api/schema/index.js';
+import type {
+  AzureContentFilter,
+  AzureFilterThreshold
+} from '../orchestration-types.js';
+
+/**
+ * Convenience function to create Azure content filters.
+ * @param filter - Filtering configuration for Azure filter. If skipped, the default Azure content filter configuration is used.
+ * @returns An object with the Azure filtering configuration.
+ * @deprecated Since 1.8.0. Use {@link buildAzureContentSafetyFilter()} instead.
+ */
+export function buildAzureContentFilter(
+  filter?: AzureContentSafety
+): InputFilteringConfig | OutputFilteringConfig {
+  if (filter && !Object.keys(filter).length) {
+    throw new Error('Filter property cannot be an empty object');
+  }
+  return {
+    filters: [
+      {
+        type: 'azure_content_safety',
+        ...(filter && { config: filter })
+      }
+    ]
+  };
+}
+
+/**
+ * Convenience function to create Azure content filters.
+ * @param config - Filtering configuration for Azure filter. If skipped, the default Azure content filter configuration is used.
```

Comment:
```suggestion
 * @param config - Configuration for Azure content safety filter. If skipped, the default configuration is used.
```

[pp] maybe mention what the default is

Reply:
The default is not documented atleast in the help portal. I referred [directly to the code](https://github.tools.sap/AI/llm-orchestration/blob/main/src/orchestration/modules/filtering/filters/azure_content_safety_service.py#L41) now.